### PR TITLE
feat : TailwindCSS and React-Router  

### PR DESCRIPTION
**Pull request by WiseyXD**
Fixed eslint errors

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,18 +10,25 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "localforage": "^1.10.0",
```
Comment (quickank1t) [inline package.json:14]: why do you need this dependency?

File: package.json [hunk 2]
```diff
@@ -10,18 +10,25 @@
...
+    "match-sorter": "^6.3.1",
```
Comment (quickank1t) [inline package.json:15]: why do you need this dependency?

File: package.json [hunk 3]
```diff
@@ -10,18 +10,25 @@
...
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
+    "react-router-dom": "^6.15.0",
+    "sort-by": "^1.2.0"
```
Comment (quickank1t) [inline package.json:19]: why do you need this dependency?

File: package.json [hunk 4]
```diff
@@ -10,18 +10,25 @@
...
     "@types/react-dom": "^18.2.7",
     "@vitejs/plugin-react": "^4.0.3",
+    "autoprefixer": "^10.4.15",
     "eslint": "^8.48.0",
     "eslint-plugin-react": "^7.32.2",
     "eslint-plugin-react-hooks": "^4.6.0",
     "eslint-plugin-react-refresh": "^0.4.3",
-    "vite": "^4.4.5"
+    "postcss": "^8.4.29",
+    "tailwindcss": "^3.3.3",
```
Comment (quickank1t) [inline package.json:34]: tailwind should be in another pr not the same..

File: src/Components/Footer.jsx [hunk 5]
```diff
@@ -0,0 +1,5 @@
+import React from "react";
```
[the remark this answers is not in the document]
Comment (quickank1t) [reply src/Components/Footer.jsx]: fix eslint issues

**Review by quickank1t — CHANGES_REQUESTED** [review]

File: index.html [hunk 6]
```diff
@@ -1,13 +1,13 @@
...
-    <script type="module" src="/src/main.jsx"></script>
-  </body>
+	<head>
+		<meta charset="UTF-8" />
+		<link rel="icon" type="image/svg+xml" href="/vite.svg" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Vite + React</title>
+	</head>
+	<body>
+		<div id="root">Not Rendered</div>
```
Comment (quickank1t) [inline index.html:10]: why do you need not rendered ?

**Review by quickank1t — CHANGES_REQUESTED** [review]

File: index.html (hunk 6 above)
Comment (WiseyXD) [reply index.html:10]: To know wether the app is getting injected into the rrot or not

File: package.json (hunk 3 above)
Comment (WiseyXD) [reply package.json:19]: These were all the dependencies mentioned in the react router docs 

You can refer the docs here : https://reactrouter.com/en/main/start/tutorial

File: package.json (hunk 2 above)
Comment (WiseyXD) [reply package.json:15]: These were all the dependencies mentioned in the react router docs

You can refer the docs here : https://reactrouter.com/en/main/start/tutorial

File: package.json (hunk 1 above)
Comment (WiseyXD) [reply package.json:14]: These were all the dependencies mentioned in the react router docs

You can refer the docs here : https://reactrouter.com/en/main/start/tutorial

**Comment by WiseyXD** [comment]
Sending in new pull request

**Comment by quickank1t** [comment]
changes are not working.

Screenshot 2023-09-06 at 08 52 48

also when i request any change you cannot resolve it, just comment it and i will decide if i have to resolve it or not.

also, i requested you to do two prs for two changes.. still i see only one for both tailwind and route

**Comment by quickank1t** [comment]
closing this pr. make two PRs one for tailwind and the other for routes with Clark. and not in the fork. clone this branch only and do changes in multiple branches.

**Pull request closed without merging**